package.json

```diff
@@ -1,41 +1,37 @@
 {
```

Comment:
Yea that's something that really drives me crazy about npm. On the one hand they say "package.json is for machines - no comments and no formatting allowed." On the other they encourage the community to keep writing ever more complicated `npm run` scripts. I read the package.json of my project more than any other single file probably. Anyway, that's why I like the formatting.

More specifically, if you're adding `lodash.has` feel free to just pull `lodash`. Since this is server side there really isn't a bundle size constraint and with lodash now the most popular project on npm it's pretty unlikely that this will introduce a new dep for anybody.